AdminCore.Constants/AzureRoleEnumConvert.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using AdminCore.Constants.Enums;
+
+namespace AdminCore.Constants
+{
+  public static class AzureRoleEnumConvert
+  {
+    public static EmployeeRoles ConvertToEmployeeRoles(string azureRole)
```

Comment:
is this behaviour tested?

Reply:
Tests not written for this logic. Uses attributes instead.